packages/next/package.json

```diff
@@ -70,6 +70,7 @@
   },
   "dependencies": {
     "@next/env": "12.1.7-canary.7",
+    "edge-runtime": "1.0.0",
```

Comment:
Can we make this a `devDependency` and pre-compile it via our `taskfile` e.g. https://github.com/vercel/next.js/blob/9f0024a5ee7bb06512aa401f1cd9726658da242e/packages/next/taskfile.js#L131

Currently this is increasing the install size by  `13.2 MB`

Reply:
Hey, I reduced the initial bundle from 13 MB to 5 MB 🎉

about create the pre-compile version: "edge-runtime" is using "@edge-runtime/primitives", and it's also a dependency. How that should be treated?